stirling-pdf/src/main/resources/templates/fragments/navElements.html

```diff
@@ -236,7 +239,10 @@
       </div>
       <div
         th:replace="~{fragments/navbarEntry :: navbarEntry('unlock-pdf-forms', 'preview_off', 'home.unlockPDFForms.title', 'home.unlockPDFForms.desc', 'unlockPDFForms.tags', 'other')}">
-      </div>  
+      </div>
+      <div
+        th:replace="~{fragments/navbarEntry :: navbarEntry('add-attachments', 'attachment', 'home.attachments.title', 'home.attachments.desc', 'attachments.tags', 'other')}">
```

Comment:
Missing translations for this?

Reply:
They should be here: https://github.com/Stirling-Tools/Stirling-PDF/pull/3781/files#diff-93a3789ddf8d5cc4892d099d90dc7e740bdf6964adeb84ab5464c024c9a942c4